packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -2457,62 +2457,97 @@ export class ProjectService extends BaseService {
             description: 'Gets a single explore from the cache',
```

Comment:
will review this soon! When you get latest Sentry, could you change this `Sentry` work here? Should be similar to: 
https://github.com/lightdash/lightdash/blob/main/packages/backend/src/services/ProjectService/ProjectService.ts#L2452

```ts
 return Sentry.startSpan(
            {
                op: 'ProjectService.getExplore',
                name: 'ProjectService.getExplore',
            },
            async () =>
                wrapOtelSpan('ProjectService.getExplore', {}, async () => {
....

```

no need to `.finish()` anymore as well

Reply:
done ✅